src/pages/contacts.tsx

```diff
@@ -0,0 +1,136 @@
+import styled from "@emotion/styled";
+import { Logo } from "../components/Logo";
+import { Menu } from "../components/Menu";
+import { PageProps } from "gatsby";
+import { Button, TextField } from "@mui/material";
+import { useState } from "react";
+import { sendTelegramMessage } from "../server/sendToTelegram";
+import { StaticImage } from "gatsby-plugin-image";
+
+const Contacts: React.FC<PageProps> = () => {
+	const [name, setName] = useState('')
+	const [email, setEmail] = useState('')
+	const [phone, setPhone] = useState('')
+	const [message, setMessage] = useState('')
+
+	const [errorName, setErrorName] = useState(false)
+	const [errorEmail, setErrorEmail] = useState(false)
+	const [errorMessage, setErrorMessage] = useState(false)
+	const [isVisible, setIsVisible] = useState(true)
+
+	const convertMessage = () => {
+		if (!name || !email || !message) return
+		const text = name + ' отправил(а) сообщение через сайт:\n' + message + '\nКонтакты для связи:\nEmail: ' + email + (phone && ('\nТелефон: ' + phone))
+		sendTelegramMessage(text)
+		setIsVisible(false)
+	}
+
+	return (
+		<Wrapper>
+			<Logo />
+			<Menu />
+			<ContactsWrap>
+				{isVisible && <ContactsInfo>
+					<Title>Contacts:</Title>
+					<Contact>
+						<SVGWrapper><svg xmlns="http://www.w3.org/2000/svg" width="25px" height="25px" viewBox="0 0 512 512"><path d="M64 112c-8.8 0-16 7.2-16 16v22.1L220.5 291.7c20.7 17 50.4 17 71.1 0L464 150.1V128c0-8.8-7.2-16-16-16H64zM48 212.2V384c0 8.8 7.2 16 16 16H448c8.8 0 16-7.2 16-16V212.2L322 328.8c-38.4 31.5-93.7 31.5-132 0L48 212.2zM0 128C0 92.7 28.7 64 64 64H448c35.3 0 64 28.7 64 64V384c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V128z" /></svg></SVGWrapper>
+						<Subtitle>Email:</Subtitle>
+						<Subtitle>pete4kinone@gmail.com</Subtitle>
+					</Contact>
+					<Contact>
+						<SVGWrapper><svg xmlns="http://www.w3.org/2000/svg" width="25px" height="25px" viewBox="0 0 384 512"><path d="M16 64C16 28.7 44.7 0 80 0H304c35.3 0 64 28.7 64 64V448c0 35.3-28.7 64-64 64H80c-35.3 0-64-28.7-64-64V64zM224 448a32 32 0 1 0 -64 0 32 32 0 1 0 64 0zM304 64H80V384H304V64z" /></svg></SVGWrapper>
+						<Subtitle>Phone:</Subtitle>
```

Comment:
It's bad that text is hardcored
But I made it same way, so it will be fixed in other time (hope so)

Reply:
@jathpr what does this mean exactly?^^"

Reply:
prefer to get all data in you code programmaticaly
like: `<Subtitle>{someDataHolder.contacts.phone}</Subtitle>`